node/middlewares/userSitemap.ts

```diff
@@ -0,0 +1,24 @@
+import * as cheerio from 'cheerio'
+import {map, values} from 'ramda'
+import Routes from '../resources/routes'
+
+export const userSitemap = async (ctx: Context) => {
+  const routes = new Routes(ctx.vtex, {timeout: 2000})
+  const userRoutes = values((await routes.getUserRoutes())['vtex.admin-pages'])
```

Comment:
what happens here if the request fails or returns null ?

Reply:
Changed so that if the request returns 404 or null (which I think will never happen), we generate an empty sitemap. Any other error is thrown.